src/styles/_header.scss

```diff
@@ -37,5 +37,9 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

Comment:
same here

Reply:
![image](https://user-images.githubusercontent.com/92334519/182962759-cc9dbf43-d42e-455f-b2ef-407d736df45e.png)
![image](https://user-images.githubusercontent.com/92334519/182962819-4606ed8c-5f10-40e4-aa09-a30898362ef7.png)
Everything was added in first commit, check it please

Reply:
https://roman-malysh.github.io/layout_catalog_hovers/